Source/Scene/Scene.js

```diff
@@ -597,7 +597,7 @@ define([
          */
         pickPositionSupported : {
             get : function() {
-                return this._context.depthTexture;
+                return defined(this._context.depthTexture) && defined(this._globeDepth);
```

Comment:
You are correct, I was thrown off by the name it should probably be something like  `hasDepthTexture`.  At one point I thought the `_globeDepth` check was needed, but I might have been thrown off by the check on line 1839.  I removed it and it looks like everything still passes.